linchpin/api/__init__.py

```diff
@@ -188,6 +188,8 @@ def run_playbook(self, pinfile, targets='all', playbook='up'):
         if self.ctx.cfgs.get('ansible'):
             ansible_console = ast.literal_eval(self.ctx.cfgs['ansible'].get('console', 'False'))
 
+        self.ctx.evars['creds_path'] = self.ctx.creds_path
```

Comment:
Use set_evar here. Failed CI due to the value not being there. set_evar makes sure to create the structure for you if it isn't there.